.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,27 @@
-<!--
-Checklist for submitting a Pull Request
+<!-- PLEASE REMOVE THIS COMMENT BLOCK BEFORE SUBMITTING
 
-Please remove this comment block before submitting.
+Thanks for sending a pull request!  Before hitting the button, please read
+this.
 
-1. Please read our [contributor guidelines](https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md).
-2. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
-3. If you want this PR to automatically close an issue when it is merged,
+1. Please make sure this comment describes:
+  - What you did
+  - Why you did it
+  - Things to consider when reviewing
+2. Read our contributor guidelines: https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md
+3. See our developer guide: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md
+4. If you want this PR to automatically close an issue when it is merged,
    add `fixes #<issue number>` or `fixes #<issue number>, fixes #<issue number>`
    to close multiple issues (see: https://github.com/blog/1506-closing-issues-via-pull-requests).
-4. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.
+5. Follow the instructions for writing a release note: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes
```

Comment:
I think this should be removed from the general template. The average contributor doesn't have access to set labels.